### ISSUE TITLE
Make dojo version of the lamp

### ISSUE THREAD

**Issue by kentcdodds** [kentcdodds/genie#16]
Jon Stolte may help with this!

**Comment by jstolte** [comment]
Unless someone else is doing this… great – they probably have more time than I do… however, I still intend to play with it throughout our monthly Innovation days until I have something that I can functional demonstrate in our product.

-          December ’14 we did not have an Innovation day as too many were scheduling time off around the Holidays.

-          January ’14 we did not have an Innovation day as everyone was focused on a Major product version release.

-          I have the base genie.js code getting loaded appropriately.

-          I have a place holder for a Dojo based Genie lamp… just trying to figure out how the Angular lamp works so I can duplicate as close as possible with a Dojo Widget.

-          The place-holder object is currently getting tickled with Ctrl-Space or by clicking on a logo icon that is not used for anything other than the logo presence.

-          Progress is slow as I only get a few hours a month to really concentrate on development

Cheers,
Jon
